fiftyone/core/stages.py

```diff
@@ -8168,7 +8168,9 @@ def load_view(self, sample_collection, reload=False):
             name = None
 
         try:
-            last_dataset = fod.load_dataset(name)
+            # Always reload the main dataset, the `reload` parameter is for
+            #   reloading patches dataset
+            last_dataset = fod.load_dataset(name, reload=True)
```

Comment:
I'm concerned setting this true will cause notable performance issues/slowdowns

Reply:
Can you explain please, I didn't fully understand what you were saying earlier

Reply:
Reloading the patches dataset sounds like potentially regenerating the patches collection or otherwise syncing the fields, which could make loading take unexpectedly long and in many cases is an unnecessary bottleneck... but I haven't gotten a chance to test/dig into the code so I guess just ignore for now...

Reply:
yeah its just going to reload the base dataset if necessary. The logic below determines whether it should regenerate patches collection I think.